### PR TITLE
"Fixed-entities-helper"

### PR DESCRIPTION
**Pull request by MallanagoudaB**

### CODE REVIEW

File: src/module/entities/helper.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ const entityTypesHelper = require(MODULES_BASE_PATH + '/entityTypes/helper')
 // const userRolesHelper = require(MODULES_BASE_PATH + "/userRoles/helper");
 // const userProfileService = require("../../generics/");
 const entitiesQueries = require(DB_QUERY_BASE_PATH + '/entities')
+const entityTypeQueries = require(DB_QUERY_BASE_PATH + '/entityTypes')
 // const formService = require(PROJECT_ROOT_DIRECTORY + '/generics/services/form')
```
Comment (VISHNUDAS-tunerlabs) [inline src/module/entities/helper.js:15]: @MallanagoudaB remove these if not used

Comment (MallanagoudaB) [reply src/module/entities/helper.js:15]: we wan this one

Comment (VISHNUDAS-tunerlabs) [reply src/module/entities/helper.js:15]: @MallanagoudaB remove the commented imports

File: src/module/entities/helper.js [hunk 2]
```diff
@@ -566,7 +566,7 @@ module.exports = class UserProjectsHelper {
 		return new Promise(async (resolve, reject) => {
 			try {
 				// Find the entities document based on the entityType in queryParams
-				let entitiesDocument = await entitiesQueries.findOne({ entityType: queryParams.type }, { _id: 1 })
+				let entitiesDocument = await entityTypeQueries.findOne({ name: queryParams.type }, { _id: 1 })
```
Comment (VISHNUDAS-tunerlabs) [inline src/module/entities/helper.js:569]: @MallanagoudaB make it entityTypeDocument

File: src/module/entities/helper.js [hunk 3]
```diff
@@ -767,10 +767,10 @@ module.exports = class UserProjectsHelper {
 				// }
 
 				// Find the entity type document based on the provided entityType
-				let entitiesDocument = await entitiesQueries.findOne(
+				let entitiesDocument = await entityTypeQueries.findOne(
 					// let entityTypeDocument = await database.models.entityTypes.findOne(
```
Comment (VISHNUDAS-tunerlabs) [inline src/module/entities/helper.js:771]: @MallanagoudaB remove this

File: src/module/entities/helper.js [hunk 4]
```diff
@@ -767,10 +767,10 @@ module.exports = class UserProjectsHelper {
 				// }
 
 				// Find the entity type document based on the provided entityType
-				let entitiesDocument = await entitiesQueries.findOne(
+				let entitiesDocument = await entityTypeQueries.findOne(
```
Comment (VISHNUDAS-tunerlabs) [inline src/module/entities/helper.js:770]: @MallanagoudaB rename the variable to entityTypeDocument

**Pull request merged**